src/tests/FavoriteArticles.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Enzyme, { shallow } from 'enzyme';
```

Comment:
You can export the enzyme package into a different file called `setupEnzyme.js`
that can be used globally in all files instead of importing the package in every file like you did here and the other test files.